client/src/components/App.js

```diff
@@ -1,36 +1,29 @@
-import React, { Component } from "react";
```

Comment:
Not a big deal but it looks like we might be using different ESLint rules.

Reply:
It does and I can not figure out why it is. I think Rob does have the elint file I pushed up. @cpdis does yours have the Elint file I merged on the first week?

Reply:
I was missing an eslint plugin.  Maybe that was causing it.  I added it today.